tree/ntuple/v7/inc/ROOT/RNTupleMerger.hxx

```diff
@@ -16,6 +16,7 @@
 #ifndef ROOT7_RNTupleMerger
 #define ROOT7_RNTupleMerger
 
+#include "ROOT/RPage.hxx"
```

Comment:
```suggestion
#include <ROOT/RPage.hxx>
```

However, what it seems you really need is `ROOT/RPageAllocator.hxx`, or better yet just a forward-declaration of `ROOT::Internal::RPageAllocator`